Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionForcePointFirewall.yaml

```diff
@@ -0,0 +1,473 @@
+                    DvcOriginalAction = DeviceAction,
+                    SrcBytes = SentBytes,
+                    DstBytes = ReceivedBytes,
+                    EventOriginalSubType = DeviceFacility,
+                    DvcId = DeviceExternalID,
+                    DvcInboundInterface = DeviceInboundInterface,
+                    DvcOutboundInterface = DeviceOutboundInterface,
+                    DvcIpAddr = DeviceAddress,
+                    EventOriginalSeverity = LogSeverity,
+                    DvcDomain = Computer        
+            | extend
+                    EventStartTime = todatetime(ReceiptTime),
+                    EventEndTime = todatetime(ReceiptTime),
+                    boolHasIP = iif(indexof_regex(requestURL,@"(\d+.\d+.\d+.\d+)") == -1, false, true),
+                    boolHasPort = iif(requestURL has ':', true, false)
+            | extend 
+                     NetworkRuleName = coalesce(DeviceCustomString1, //RuleId
+                                                DeviceCustomString2), //NatRuleId
+                     ThreatId = coalesce(DeviceCustomString3, //VulnerabilityReferences
+                                        DeviceCustomString4), //VirusId
+                     DstDomain = case(
+                                    boolHasPort and not(boolHasIP), tostring(split(requestURL,':',0)[0]),
+                                    not(boolHasIP), requestURL,
+                                    ""
+                                ),
+                     DstIpAddr = case(
+                                    isnotempty(DstIpAddr), DstIpAddr,
+                                    not(boolHasIP), requestURL,
+                                    ""
+                                )
+            | extend
+                     DvcIdType = case(
+                                    isnotempty(DvcId), "Other",
+                                    ""
+                                ),
+                     DvcDomainType = case (
+                                    isnotempty(DvcDomain), "FQDN",
+                                    ""
+                                ),
+                     DstDomainType = case(
+                                    isnotempty(DstDomain), "FQDN",
+                                    ""  
+                                ),
+                     DstPortNumber = case(
+                                    isnotempty(DstPortNumber), DstPortNumber,
+                                    ApplicationProtocol startswith "TCP", toint(split(ApplicationProtocol,'/')[1]),
+                                    ApplicationProtocol startswith "UDP", toint(split(ApplicationProtocol,'/')[1]),
+                                    boolHasPort, toint(split(requestURL,':',0)[1]),
+                                    int(null)
+                            ),
+                     AdditionalFields = pack("RequestMethod",iff(isnotempty(RequestMethod) and RequestMethod != "UNKNOWN", RequestMethod, ""),
```

Comment:
If RequestMethod is available, probably a WebSession

Reply:
Added more information in requestURL comment above.